charts/bulk-scan-processor/values.aat.template.yaml

```diff
@@ -5,3 +5,4 @@ java:
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
   environment:
     STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    DELETE_COMPLETE_FILES_CRON: "0/10 * * * * *"
```

Comment:
this applies only to staging pod and not to all aat pods

Reply:
@nitinprabhu should it be in values.yaml? in this case it will also be in prod - right?

Reply:
for prod we can override in flux yaml if the value needs to be different. May be worth checking with team what we need.

I would day every 10 mins on prod and other environment 10 secs ?

Reply:
I'd make sure flux for prod says every 10 minutes before this PR gets merged.